include/vcpkg/base/messages.h

```diff
@@ -2091,5 +2091,168 @@ namespace vcpkg
                     (msg::path),
                     "",
                     "Failed to parse {path}, expected a top-level object.");
-
-}
+    DECLARE_MESSAGE(FailedToParseSerializedManifest,
+                    (msg::path),
+                    "",
+                    "Failed to parse serialized manifest file of {path} Please open an issue at "
+                    "https://github.com/microsoft/vcpkg, with the following output:");
+    DECLARE_MESSAGE(MismatchedSerializedManifestSCF,
+                    (),
+                    "",
+                    "The serialized manifest SCF was different from the original SCF. Please open an issue at "
+                    "https://github.com/microsoft/vcpkg, with the following output:");
+    DECLARE_MESSAGE(PortBugIncludeDirInCMakeHelperPort,
+                    (),
+                    "",
+                    "The folder /include exists in a cmake helper port; this is incorrect, since only cmake "
+                    "files should be installed");
+    DECLARE_MESSAGE(PortBugMissingIncludeDir,
+                    (),
+                    "",
+                    "The folder /include is empty or not present. This indicates the library was not correctly "
+                    "installed.");
+    DECLARE_MESSAGE(PortBugRestrictedHeaderPaths,
+                    (),
+                    "",
+                    "Restricted headers paths are present. These files can prevent the core C++ runtime and "
+                    "other packages from compiling correctly:");
+    DECLARE_MESSAGE(PortBugAllowRestrictedHeaders,
+                    (msg::env_var),
+                    "",
+                    "In exceptional circumstances, this policy can be disabled via {env_var}");
```

Comment:
I think these messages should be combined, as in:

```suggestion
    DECLARE_MESSAGE(PortBugRestrictedHeaderPaths,
                    (msg::env_var),
                    "A list of restricted headers is printed after this message, one per line.",
                    "The following restricted headers can prevent the core C++ runtime and other packages from compiling correctly. In exceptional circumstances, this policy can be disabled via {env_var}.");
```